jenkins-pipelines/Jenkinsfile

```diff
@@ -46,16 +48,19 @@ pipeline {
                     git url:"${INFRASTRUCTURE_REPOSITORY}"
                 }
 
+                // curl ${TESTGRID_DIST_URL} -o ${TESTGRID_NAME}.zip
+                // cp /testgrid/jenkins-home/workspace/${TESTGRID_NAME}-${TESTGRID_VERSION}.zip .
```

Comment:
I think we can remove this two lines now.. :)

Reply:
Will fix this :)